src/styl/_components/_inlinemenu.styl

```diff
@@ -0,0 +1,51 @@
+// Inline Menu
+//
+// Set all elements of list in display flex, justify and align items center.
+//
+// This class can be used to make a inline menu.
+//
+// Markup: inlinemenu.twig
+//
+// Styleguide: Components.inlinemenu
+
+.menu
+    &-list
+        _pl(0)
+        display flex
+        justify-content center
+        align-items center
+        list-style-type none
+        white-space nowrap
+        width 100%
+
+        @media $breakpoints.md
+            _m(1 auto)
+            width 64%
+            white-space normal
+            flex-wrap wrap 
+
+        @media $breakpoints.lg
+            _m(1 auto) 
+            width 100% 
+            white-space normal 
+            flex-wrap wrap 
+            
+        .list-elemt 
```

Comment:
We keep the same SMACSS logic, so this should be `.menu-list-elemt` and not `.menu-list .list-elemt`
Also, it might be a force of habit but I would prefer to name this `item` (instead of elemt)